### PR TITLE
[player-2011]

### PR DESCRIPTION
**Pull request by aeng7**
-added new quality selection config

### CODE REVIEW

File: skin.json [hunk 1]
```diff
@@ -209,6 +209,9 @@
     },
     "tooltips":{
       "enabled":false
+    },
+    "qualitySelection":{
```
Comment (jpfranco) [inline skin.json]: Can we add this to the schema?

Comment (aeng7) [reply skin.json]: Where is the schema located?

Comment (jpfranco) [reply skin.json]: It's the `skin-schema.json` file. It's used for docs and validation.

Comment (aeng7) [reply skin.json]: Done!

File: skin-schema.json [hunk 2]
```diff
@@ -689,6 +689,17 @@
...
             }
           }
+        },
+        "qualitySelection":{
+          "description":"Configuration for the control bar quality selection panel/popover",
+          "type":"object",
+          "properties":{
+            "format":{
+              "description":"Determine the format of the quality options. Available formats: bitrate only, resolution only, and resolution + bitrate. Comma or white space separated values of: \'bitrate\' and \'resolution\'",
+              "type":"string",
```
Comment (jpfranco) [inline skin-schema.json]: I think it could be useful to add `enum` here in order to limit the setting to valid values.

Comment (aeng7) [reply skin-schema.json]: Added

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**